### DISCUSSION TITLE
Detecting if a REPL session is active

### DISCUSSION THREAD

**Discussion by brunnelu** [micropython/micropython#10496] [Using MicroPython] [answered]
HI there,

I would like to know if there is a way to detect if a REPL session is active in my micropython code. On the web I did not find anything about it. I guess this is to much of an edge case.

This would be useful to terminate code run from the autostart. 

Thank you in advance

**Comment by dlech** [comment, marked as answer]
Similar discussion: https://github.com/micropython/micropython/discussions/9245

**Reply by brunnelu** [reply]
Oh Thanks. 
Sorry. Did not see this result.
